go-controller/pkg/node/gateway_localnet.go

```diff
@@ -116,32 +122,107 @@ func generateGatewayNATRules(ifname string, ip net.IP) []iptRule {
 	return rules
 }
 
-func localnetGatewayNAT(ipt util.IPTablesHelper, ifname string, ip net.IP) error {
-	rules := generateGatewayNATRules(ifname, ip)
-	return addIptRules(ipt, rules)
+func localnetGatewayNAT(localNetdata []*localnetData, ifname string) error {
+	for _, lnData := range localNetdata {
+		rules := generateGatewayNATRules(ifname, lnData.gatewayIP)
+		err := addIptRules(lnData.ipt, rules)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
 }
 
-func initLocalnetGateway(nodeName string, subnet *net.IPNet, wf *factory.WatchFactory, nodeAnnotator kube.Annotator) error {
-	// Create a localnet OVS bridge.
+func initLocalnetGateway(nodeName string, subnets []*net.IPNet, wf *factory.WatchFactory, nodeAnnotator kube.Annotator) error {
+
```

Comment:
(Don't start functions with a blank line. Except when the function header is wrapped onto a second line and gets indented and so it's hard to read without adding a blank line.)

Reply:
Will do. I thought go formatting will fix all these which I run before committing.

Reply:
The "where blank lines do and don't belong" rules are kind of vague and not universally consistent, so gofmt doesn't get involved with that.

Likewise for the import ordering; there are a few different styles. Some people split "local" (same-project) imports out into a separate group, some don't, etc.